src/canvas.rs

```diff
@@ -97,6 +127,46 @@ impl<P: AsBytes + FromBytes> Canvas<P> {
         Self::with_layout(layout)
     }
 
+    /// Interpret an existing buffer as an image.
```

Comment:
Shouldn't this be "as a *Canvas*"?

Reply:
This is `Canvas` afterall but I agree it should not refer to an 'image'.